### ISSUE TITLE
[bug] Tamagui starter fails to run on Android: java.lang.Double cannot be cast...

### ISSUE THREAD

**Issue by ArcadeCityMayor** [tamagui/tamagui#472]
**Describe the bug**

A fresh Tamagui app via `npm create tamagui` fails to run on Android device or emulator, giving the following error. 

Looks like it [may be related](https://stackoverflow.com/questions/74189191/java-lang-double-cannot-be-cast-to-com-facebook-react-bridge-readablemap-error-s) to the react-native-svg version, but the suggested version (12.1.0) is already the only one in yarn.lock. Unsure how to fix.

![androidbug](https://user-images.githubusercontent.com/14167547/210107384-0cb8f1bc-5fda-42a8-a03b-0a4b67cbb888.png)

**To Reproduce**

```
npm create tamagui@latest [or npm create tamagui]
cd appdir
yarn native
Press {a} to run on Android emulator or device
```

**System Info (please complete the following information):** 

```
  System:
    OS: macOS 13.0.1
    CPU: (16) x64 Intel(R) Xeon(R) W-2140B CPU @ 3.20GHz
    Memory: 54.34 MB / 32.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 14.19.3 - ~/.nvm/versions/node/v14.19.3/bin/node
    Yarn: 1.22.19 - ~/.yarn/bin/yarn
    npm: 6.14.17 - ~/.nvm/versions/node/v14.19.3/bin/npm
  Browsers:
    Chrome: 108.0.5359.124
    Safari: 16.1
```

Received same error on emulator (Pixel 3 XL API 30) and device (Galaxy S9).

Error goes away when I uninstall react-native-svg and the tamagui icons package and comment out the references.

**Comment by natew** [comment]
I had that issue due to react-native-svg, but yea I fixed it by pinning to 12.1.0 so that's an odd error.. I had luck with 12.13 I think too.

**Comment by nickreese** [comment]
Getting this as well.

**Comment by lanceturbes** [comment]
I receive the same error when starting up a new project with the starter template. 

My `package.json` has `react-native-svg` set to `12.1.0` as well (came like that out of the box).

**Comment by natew** [comment]
Fix in [ecd81b5](https://github.com/tamagui/tamagui/pull/480/commits/ecd81b5ae3738899b23c31f06632ab4601f4fcc0) should be merged today.

Just upgrade to 13.4.0 in the meantime.

**Comment by ArcadeCityMayor** [comment]
Tested the fix in updated starter, it works. Closing
